From: Matthias Fehring <buschmann23@opensuse.org>
Date: 2022-01-25 19:36:00 +0100
Subject: skip acl tests on old servers
Upstream: no

Redis server prior to version 6.2.0 fail executing some ACL specific tests.
According to Redis changelog, some ACL support has been introduced with
version 6.2-rc1.

---
 test_commands.py |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/tests/test_commands.py	2022-01-25 19:34:59.427241656 +0100
+++ b/tests/test_commands.py	2022-01-25 19:33:52.555242854 +0100
@@ -104,7 +104,7 @@ class TestRedisCommands:
         password = await r.acl_genpass()
         assert isinstance(password, str)
 
-    @skip_if_server_version_lt(REDIS_6_VERSION)
+    @skip_if_server_version_lt("6.2.0")
     async def test_acl_getuser_setuser(self, r: aioredis.Redis, request, event_loop):
         username = "redis-py-user"
 
@@ -219,7 +219,7 @@ class TestRedisCommands:
         )
         assert len((await r.acl_getuser(username))["passwords"]) == 1
 
-    @skip_if_server_version_lt(REDIS_6_VERSION)
+    @skip_if_server_version_lt("6.2.0")
     async def test_acl_list(self, r: aioredis.Redis, request, event_loop):
         username = "redis-py-user"
 
